lib/matplotlib/ticker.py

```diff
@@ -958,10 +974,17 @@ def __call__(self, x, pos=None):
         sign = np.sign(x)
         # only label the decades
         fx = math.log(abs(x)) / math.log(b)
+
         isDecade = is_close_to_int(fx)
+        exponent = np.round(fx) if is_decade else np.floor(fx)
```

Comment:
Is it normal to use `is_decade` here rather than the `isDecade` define just before? Besides, there is also a _function_ called `is_decade` at the module level (see l. 1781).